pkgs/development/tools/build-managers/cmake/default.nix

```diff
@@ -58,6 +61,7 @@ stdenv.mkDerivation rec {
       "--no-system-jsoncpp"
     ]
     ++ optional (!stdenv.isCygwin) "--system-libs"
```

Comment:
That's what I was thinking, and it looks like I missed `--no-system-libarchive`, if I use that I can get rid of openssl without most of #21101